doc/modules/language-guide/examples/CounterCertified.mo

```diff
@@ -0,0 +1,37 @@
+/// Simple counter (see `Counter.mo`), but uses `mo:base/CertifiedData` to
+/// implement the counter value as a certified variable.
+import CD "mo:base/CertifiedData";
+import Blob "mo:base/Blob";
+import Nat8 "mo:base/Nat8";
+import Nat32 "mo:base/Nat32";
+
+actor Counter {
+
+  var value : Nat32 = 0;
+
+  /// Helper; should be in base?
```

Comment:
You can use Claudio’s new stable memory functions for this :-D

Reply:
Sure, but that's yet another experimental feature.

And perhaps just as important, I've been told repeatedly that these features are to be thought of as orthogonal, so we best not show that even our simplest examples mix them, lest we confuse people about how they interact?

FWIW, no strong feeling on my part about this.